addon/helpers/find-by.js

```diff
@@ -29,7 +29,7 @@ export default Helper.extend({
       let array = get(this, 'array');
       let value = get(this, 'value');
 
-      return array.findBy(byPath, value);
+      return array.find(e => get(e, byPath) === value);
```

Comment:
Out of curiosity, why is that preferable? (Unless this addons targets 1.13 and ie8)